docker/README.md

```diff
@@ -5,11 +5,19 @@ You may build this via
 docker build --tag aerogear.org .
 ```
 
+To run the container:
+
+```
+./run.sh
+```
+from this directory (does not work if you run it from a different directory).
+
+
```

Comment:
@secondsun wanna keep run.sh simple, but you're right, it doesn't flow, will add 'Advanced' section.